### PR TITLE
l10n part 1 - taxes and conversion

### LINKED ISSUE #6: Use user location to determine tax

### PR DESCRIPTION
**Pull request by dtjanaka**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -2,6 +2,8 @@
 //TODO: account for website discounts
 //TODO: use user location to calculate local tax and display local currency
 
+const allTaxRates = require('./sales_tax_rates.json');
```
Comment (matthewcn56) [inline script.js]: Do you need to include the minified package from [require.js](https://requirejs.org/docs/api.html#jsfiles) to allow require? Or else we will have to switch to a node environment and can't just serve a static html site

Comment (matthewcn56) [reply script.js]: I'm getting errors on my end from the console, stopping loading.
Screen Shot 2021-08-05 at 11 36 44 PM

**Review by matthewcn56 — CHANGES_REQUESTED** [review]
This looks really great! However, I'm getting errors from our require of our sales_tax_rates json object. Since we're just serving a static html page instead of working in a node environment, we'll need the minified files of [require.js](https://requirejs.org/) for in-browser file-loading or we'll have to change the format that we bring the .json object in, just making it into a .js file and importing It in normally.

**Comment by dtjanaka** [comment]
ah i see, i'll just make it a regular js file

**Comment by dtjanaka** [comment]
ah wait the object returned is a little more complex than I remembered; i'll fix this tmrw

**Comment by dtjanaka** [comment]
@matthewcn56 this should be ready to merge now, no display changes yet but it pulls the country (and state if applicable) and also gets the tax rate. You can see a live version at https://boba2.dylon.us/ in the console.

**Review by matthewcn56 — APPROVED** [review]
Looks great! Now it properly loads in even if location services are disabled as well

**Pull request merged**